extern/jsoncpp/src/lib_json/json_reader.cpp

```diff
@@ -877,8 +877,7 @@ std::istream& operator>>( std::istream &sin, Value &root )
     Json::Reader reader;
     bool ok = reader.parse(sin, root, true);
     //JSON_ASSERT( ok );
-//    if (!ok) throw std::runtime_error(reader.getFormatedErrorMessages());
-    assert(ok);
```

Comment:
This change was not undone in the original version of this PR.
It is more than 10 years old with no comment: https://github.com/stepmania/stepmania/blame/beb793267bfd3927a3bc801c0f03251e0e26caac/src/jsoncpp/src/lib_json/json_reader.cpp

Probably it was introduced to avoid exceptions.
As assert calls abort on failure, the behavior of crashing by not catching this exception should be the same.